src/Common/src/Interop/Interop.RichEditMessages.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    public static class RichEditMessages
```

Comment:
Might want to add a warning/comment here. If the conflicting messages are removed like @JeremyKuhne suggested this is probably not necessary though.

```suggestion
    /// <summary>RichTextBox specific window messages. Note that some messages have the same name but different value compared to normal TextBox messages.</summary>
    public static class RichEditMessages
```